### ISSUE TITLE
Dll error

### ISSUE THREAD

**Issue by tnbair** [FoxAhead/Far-Cry-2-Multi-Fixer#7]
BytesRead 512

BytesWritten 273

BytesWritten 13

Error: Dll not loaded

i get this error, is it possible to fix

**Comment by FoxAhead** [comment]
Yes. Make sure you are not using some pirate version. Use official game release from digital stores. Check if your anti-malware software is blocking MultiFixer. Run as admin. Try installing the game to different simple path like C:\FarCry2

https://github.com/FoxAhead/Far-Cry-2-Multi-Fixer/issues/4#issuecomment-993683797
